### PR TITLE
Fixed some issues related to Scoreboards

### PR DESCRIPTION
**Pull request by Tim203**

### CODE REVIEW

**Review by lifegame1lu111 — COMMENTED** [review]
Seems pretty nice the fix!

**Comment by DanielKaleby** [comment]
Has that been fixed? I didn't understand it very well, sorry, I'm a bit of a layman in these matters.

If yes (for "has it been fixed?") My server is on Aternos, do you have any "step by step" of what to do?

**Comment by DoctorMacc** [comment]
Ah; you will have to wait until this gets "merged". You'll see this turn from "open" to "merged".

**Review by DoctorMacc — APPROVED** [review]

**Pull request merged**
